core/services/functions/listener.go

```diff
@@ -366,21 +348,49 @@ func (l *FunctionsListener) handleOracleRequest(request *ocr2dr_oracle.OCR2DROra
 		return
 	}
 
-	run := pipeline.NewRun(*l.job.PipelineSpec, vars)
+	requestData, cborParseErr := cbor.ParseDietCBOR(request.Data)
```

Comment:
@justinkaseman if you rebase #8769 on top of this PR, you'll be able to extract aggregation method from requestData here.

@KuphJr similarly you can extract secrets URL here.